1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,22 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+const quoteDiv = document.getElementById("quotes");
+
+const ourQuote = document.createElement("p");
+
+const randomQuote = pickFromArray(quotes);
+
+// console.log(randomQuote.author);
+
+ourQuote.innerHTML = `${randomQuote.quote}, said by ${randomQuote.author}`;
+
+quoteDiv.append(ourQuote);
+
+const ourButton = document.getElementById("button");
+
+ourButton.addEventListener("click", () => {
```

Comment:
Ohhh, this was so close Saf! I think you confused the `newRandomQuote` with the `quoteDiv`, hence why it wasn't changing. Here is what I added to get it all working.

```
ourButton.addEventListener("click", () => {
	const { quote, author} = pickFromArray(quotes);
	quoteDiv.innerHTML = `${author}: ${quote}`;
});
```

But again, grabbing the quote and getting the correct div is great work! Really nice job again!!! 🎉